backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/employee/EmployeeService.java

```diff
@@ -25,4 +25,8 @@ public Employee addEmployee(NewEmployee newEmployee) {
         Employee saveEmployee = newEmployee.withId(id);
```

Comment:
maybe write `Employee saveEmployee = new Employee(id, newEmployee.name)`, so you don't need the additional method in the NewEmployee Record.

Reply:
we have extra outsourced this here like daniels annotation